### ISSUE TITLE
How to call function 'el_bind_key'

### ISSUE THREAD

**Issue by lilastic** [troglobit/editline#46]
Using the `readline` library, one can have this `rl_bind_key('\t', rl_insert);` which disables `readline` from displaying files that exist on the current working directory when the user presses TAB.

I am now trying to get ride of `readline` and use the `editline` library instead. I found out function `el_bind_key` (which looks like it's the replacement for function `rl_bind_key`) but not sure how the second parameter of this function should look like.

Basically, I want to disable `editline` displaying files that exist on the current working directory when the user presses TAB.

**Comment by troglobit** [comment]
See the examples/cli.c for instance.

**Comment by lilastic** [comment]
Thanks @troglobit ! I managed to solve the issue concerning TAB with the example that you have pointed out.

However, when I press `CTRL+Z` (to suspend the process that runs `editline` - I'm in Linux), it doesn't react. Before, when using the `readline` librry everything worked as expect (i.e. the process would have been suspended and the prompt returns to the terminal). Do you know how I can fix this? Thanks!

**Comment by lilastic** [comment]
Solved!
